### PR TITLE
Schedule fedora builds

### PR DESCRIPTION
**Pull request by ochosi**
Fix using `kinit` and scheduling builds in all Fedoras (`packit build` only targets rawhide).

### CODE REVIEW

File: release.py [hunk 1]
```diff
@@ -382,6 +387,48 @@ def release_branch(args):
     msg_ok(f"Checked out a new release branch '{current_branch}'")
 
 
+def schedule_fedora_builds(repo):
+    """Schedule builds for all active Fedora releases"""
+    fedoras = [ 'rawhide', 'f35', 'f34', 'f33' ]
+    if repo == "osbuild":
+        url = "https://koji.fedoraproject.org/koji/packageinfo?packageID=29756"
+    else:
+        url = "https://koji.fedoraproject.org/koji/packageinfo?packageID=31032"
```
Comment (diaasami) [inline release.py:396]: may be use an if for osbuild, elif for osbuild-composer and the last else for `unsupported repo`

File: release.py [hunk 2]
```diff
@@ -382,6 +386,58 @@ def release_branch(args):
...
+        url = "https://koji.fedoraproject.org/koji/packageinfo?packageID=29756"
+    elif repo == "osbuild-composer":
+        url = "https://koji.fedoraproject.org/koji/packageinfo?packageID=31032"
+    else
+        url = "<unsupported repository>"
+
+    if os.path.isdir(repo) is False:
+        run_command(['fedpkg','clone',repo])
+    wd = os.getcwd()
+    os.chdir(f"{wd}/{repo}")
```
Comment (ochosi) [inline release.py:402]: Currently I use a sub-directory of the working / git directory for pulling in dist-git.

I could alternatively also just do this in `tmp` or something and delete the directory again as final step so this is more contained and less error-prone?

Comment (ondrejbudai) [reply release.py:402]: Yeah, I think that a temporary directory would be better.

File: release.py [hunk 3]
```diff
@@ -382,6 +386,58 @@ def release_branch(args):
...
+
+    for fedora in fedoras:
```
Comment (ondrejbudai) [inline release.py:404]: I think, in the future, we may want to build these in parallel to save some time.

File: release.py [hunk 4]
```diff
@@ -382,6 +386,58 @@ def release_branch(args):
...
+
+        if "completed successfully" in res:
+            msg_ok(f"Build for {fedora} done.")
+
+    os.chdir(wd)
+
+    msg_info(f"Check {url} for all {repo} builds.")
+
+
+def kinit(args):
```
Comment (ondrejbudai) [inline release.py:421]: Shouldn't this function be above `schedule_fedora_builds`? It makes more sense to me.

File: release.py [hunk 5]
```diff
@@ -454,13 +508,16 @@ def release_playbook(args, repo, api):
     if res != "skipped":
         create_release(args, api)
 
-    step(f"Merge the pull request in Fedora: https://src.fedoraproject.org/rpms/{repo}/pull-requests",
+    step(f"Are all related pull requests in Fedora: https://src.fedoraproject.org/rpms/{repo}/pull-requests",
```
Comment (ondrejbudai) [inline release.py:511]: Missing `merged`?

File: release.py (hunk 3 above)
Comment (ochosi) [reply release.py:404]: Good point, that'd be nice. asyncio?

**Pull request merged**